source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -94,7 +94,8 @@ private CalificationsDto GetEvaluatorEvaluation(CalificationsEvaluationDto evalu
             {
                 View = UserView.Evaluation,
                 Evaluation = evaluationDto,
-                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                Califications = califications.Where
+                (c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)).ToList()
```

Comment:
It's the idea, when the evaluation is 'readyfordevolution', not only the responsible can see the calification of the company, also evaluators and employee